.vscode/settings.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
I think you should exclude this file from the PR. Looks like a config file for your editor.